src/wiki/models/urlpath.py

```diff
@@ -281,12 +281,12 @@ def create_urlpath(
         if not site:
             site = Site.objects.get_current()
         article = Article(**article_kwargs)
-        article.add_revision(ArticleRevision(title=title, **revision_kwargs), save=True)
         article.save()
         newpath = cls.objects.create(
             site=site, parent=parent, slug=slug, article=article
         )
         article.add_object_relation(newpath)
+        article.add_revision(ArticleRevision(title=title, **revision_kwargs), save=True)
```

Comment:
It would be nice with an inline code-comment. I could guess why it's called in this order and why you changed it, it seems to make sense :+1: